staging/src/k8s.io/apiserver/pkg/endpoints/handlers/apply/versionconverter_test.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apply
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"sigs.k8s.io/structured-merge-diff/fieldpath"
+	"sigs.k8s.io/structured-merge-diff/typed"
+	"sigs.k8s.io/structured-merge-diff/value"
+)
+
+var kinds []schema.GroupVersionKind = []schema.GroupVersionKind{
+	gvkForVersion("v1"),
+	gvkForVersion("v2"),
+}
+
+// TestVersionConverter tests the version converter
+func TestVersionConverter(t *testing.T) {
+	tc := fakeTypeConverter{}
```

Comment:
I suspect this one specifically doesn't have to be faked, since it's pretty easy to create one.